editor/animation_track_editor.cpp

```diff
@@ -5393,7 +5393,138 @@ void AnimationTrackEditor::_edit_menu_pressed(int p_option) {
 
 			undo_redo->commit_action();
 		} break;
+		case EDIT_DUPLICATE_TRACKS: {
+			track_duplicate_select->clear();
+			TreeItem *troot = track_duplicate_select->create_item();
 
+			for (int i = 0; i < animation->get_track_count(); i++) {
+
+				NodePath path = animation->track_get_path(i);
+				Node *node = NULL;
+
+				if (root && root->has_node(path)) {
+					node = root->get_node(path);
+				}
+
+				String text;
+				Ref<Texture> icon = get_icon("Node", "EditorIcons");
+				if (node) {
+					if (has_icon(node->get_class(), "EditorIcons")) {
+						icon = get_icon(node->get_class(), "EditorIcons");
+					}
+
+					text = node->get_name();
+					Vector<StringName> sn = path.get_subnames();
+					for (int i = 0; i < sn.size(); i++) {
```

Comment:
This `i` is shadowing the one before it.

Reply:
Fixed by changing it to `j`